conf/solr/8.8.1/schema.xml

```diff
@@ -331,6 +331,7 @@
     <copyField source="coverage.Temporal" dest="_text_" maxChars="3000"/>
     <copyField source="coverage.Temporal.StartTime" dest="_text_" maxChars="3000"/>
     <copyField source="coverage.Temporal.StopTime" dest="_text_" maxChars="3000"/>
+<!--    <copyField source="creator" dest="_text_" maxChars="3000"/>-->
```

Comment:
Let's revert this Solr change.

Reply:
solr changes removed along with the example metadata block, etc.